erpnext/manufacturing/doctype/bom/bom.py

```diff
@@ -97,6 +98,11 @@ def get_item_det(self, item_code):
 
 		return item
 
+	def validate_raw_materials_table(self):
+		if self.manufacturing_type == "Process":
+			if len(self.items) > 1:
+				frappe.throw(_("Only one item for raw material is allowed"))
```

Comment:
Change the message to - 'For Process Manufacturing, only one item is allowed as Raw material'